package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "js-class-learn",
+  "version": "1.0.0",
+  "description": "",
+  "main": "gulpfile.js",
+  "dependencies": {},
+  "devDependencies": {
+    "gulp": "^3.9.0",
```

Comment:
While using gulp, add jshint or similar and lint your code, fix all the issues it will find.